ui/raidboss/data/06-ew/raid/p1n.txt

```diff
@@ -0,0 +1,80 @@
+28.5 "Gaoler's Flail Right/Gaoler's Flail Left" sync / 1[56]:[^:]*:Erichthonios:(6DA2|6DA3):/
+39.0 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:65F4:/
+46.2 "--knockback stack--" sync / 1[56]:[^:]*:Erichthonios:6D15:/
+51.5 "Pitiless Flail" sync / 1[56]:[^:]*:Erichthonios:65E5:/
+58.6 "True Holy" sync / 1[56]:[^:]*:Erichthonios:65E7:/
+
+## Third Gaoler's Flail
+70.5 "Gaoler's Flail Left/Gaoler's Flail Right" sync / 1[56]:[^:]*:Erichthonios:(6DA3|6DA2):/
+81.1 "Heavy Hand" sync / 1[56]:[^:]*:Erichthonios:65F3:/
+93.2 "Intemperance" sync / 1[56]:[^:]*:Erichthonios:65EE:/
+97.3 "--middle--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+105.1 "Intemperate Torment" sync / 1[56]:[^:]*:Erichthonios:65EF:/
+107.7 "Hot Spell/Cold Spell 1" sync / 1[56]:[^:]*:Erichthonios:(65F0|54F1):/
+116.7 "Hot Spell/Cold Spell 2" sync / 1[56]:[^:]*:Erichthonios:(6CC5|6CC6):/
+125.7 "Hot Spell/Cold Spell 3" sync / 1[56]:[^:]*:Erichthonios:(6CC5|6CC6):/
+135.3 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:65F4:/
+147.6 "Heavy Hand" sync / 1[56]:[^:]*:Erichthonios:65F3:/
+162.7 "Shining Cells" sync / 1[56]:[^:]*:Erichthonios:65E9:/
+174.9 "Aetherchain" sync / 1[56]:[^:]*:Erichthonios:65EB:/
+175.8 "Powerful Fire/Powerful Light" sync / 1[56]:[^:]*:Erichthonios:(65EC|65ED):/
+183.1 "Aetherchain" sync / 1[56]:[^:]*:Erichthonios:65EB:/
+184.0 "Powerful Fire/Powerful Light" sync / 1[56]:[^:]*:Erichthonios:(65EC|65ED):/
+
+## Beginning of Loop
+188.3 "--middle--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+
+# Fourth Gaoler's Flail
+198.7 "Aetherflail Left/Aetherflail Right" sync / 1[56]:[^:]*:Erichthonios:(6DA3|6DA2):/
+198.8 "Powerful Fire/Powerful Light" sync / 1[56]:[^:]*:Erichthonios:(65EC|65ED):/
+209.8 "Aetherflail Right/Aetherflail Left" sync / 1[56]:[^:]*:Erichthonios:(6DA2|6DA3):/
+209.9 "Powerful Fire/Powerful Light" sync / 1[56]:[^:]*:Erichthonios:(65EC|65ED):/
+214.3 "--knockback stack--" sync / 1[56]:[^:]*:Erichthonios:6D15:/
+219.6 "Pitiless Flail" sync / 1[56]:[^:]*:Erichthonios:65E5:/
+226.7 "True Holy" sync / 1[56]:[^:]*:Erichthonios:65E7:/
+232.9 "--middle--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+
+# Sixth Gaoler's Flail
+243.3 "Aetherflail Left/Aetherflail Right" sync / 1[56]:[^:]*:Erichthonios:(6DA3|6DA2):/
+243.4 "Powerful Fire/Powerful Light" sync / 1[56]:[^:]*:Erichthonios:(65EC|65ED):/
+260.8 "Slam Shut" sync / 1[56]:[^:]*:Erichthonios:65EA:/
+270.0 "Intemperance" sync / 1[56]:[^:]*:Erichthonios:65EE:/
+274.1 "--middle--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+281.7 "Intemperate Torment" sync / 1[56]:[^:]*:Erichthonios:65EF:/
+284.3 "Hot Spell/Cold Spell 1" sync / 1[56]:[^:]*:Erichthonios:(65F0|54F1):/
+293.3 "Hot Spell/Cold Spell 2" sync / 1[56]:[^:]*:Erichthonios:(6CC5|6CC6):/
+302.3 "Hot Spell/Cold Spell 3" sync / 1[56]:[^:]*:Erichthonios:(6CC5|6CC6):/
+305.6 "Gaoler's Flail Right/Gaoler's Flail Left" sync / 1[56]:[^:]*:Erichthonios:(6DA2|6DA3):/
+316.1 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:65F4:/ jump 1316.2
```

Comment:
Is this jump wrong too?

Could you verify this file through test_timeline too?

Reply:
Yea, the jump should be removed woops.

Reply:
My test_timeline checks are only showing' 8.709: Matched entry: 8.7 --sync-- (-0.009s)' when comparing https://www.fflogs.com/reports/mfgnwR6JQ9HMT3z4#fight=23&type=damage-done

Reply:
One thing to keep in mind, I didn't prioritize FFLogs test compatibility when we rewrote `test_timeline` to support the changes in log line formats. It's very possible that it's completely unusable against FFLogs for right now.

Reply:
Ah, thanks @JLGarber, I wasn't aware.

Reply:
p2n tested fine with a local log. I tried out a local p1n and it looks like it loops properly.